src/membraneWebRTC.ts

```diff
@@ -894,6 +897,7 @@ export class MembraneWebRTC {
     if (parameters.encodings.length === 0) {
       parameters.encodings = [{}];
     } else {
+      // TODO: sent MediaEvent with updated variant bitrates
```

Comment:
Sending `TrackVariantBitrates` MediaEvent here would be quite troublesome (need to split the bandwidth before sending bitrates for each of the tracks), so I decided to leave it to the bigger refactor that we mentioned.

Reply:
Acctually, would it be hard to address this issue in this PR?

Reply:
Ok, I did, but I would't call the code clean, the ambiguity of how track variant bitrates are kept (sometimes as an aggregated value, sometimes bitrate per variant) and that the function responsible for splitting bitrate is deeply nested on the call stack make it hard to do it nicely without bigger refactor (which, as far as I understand, we don't want to do in this PR).